langchain/agents/mrkl/index.ts

```diff
@@ -108,6 +121,34 @@ export class ZeroShotAgent extends Agent {
     });
   }
 
+  static asSqlAgent(
+    llm: BaseLLM,
+    toolkit: SqlToolkit,
+    args?: SqlCreatePromptArgs
+  ) {
+    const {
+      prefix = SQL_PREFIX,
+      suffix = SQL_SUFFIX,
+      inputVariables = ["input", "agent_scratchpad"],
+      topK = 10,
+    } = args ?? {};
+    const { tools } = toolkit;
+    const formattedPrefix = interpolateFString(prefix, {
+      dialect: toolkit.dialect,
```

Comment:
Could someone want this as an input variable?

Reply:
Probably not, you want the input variables for a ZeroShotAgent `LLMChain` to be just `input` and `agent_scratchpad`. Everything else should be resolved beforehand. We'll see if usage changes though